docs/.vuepress/config.js

```diff
@@ -200,13 +200,12 @@ module.exports = {
           collapsable: true,
           children: [
             'concepts/api',
+            'concepts/api_keys',
             'concepts/cli',
-            'concepts/projects',
             'concepts/flows',
+            'concepts/projects',
             'concepts/kv_store',
             'concepts/secrets',
-            'concepts/api_keys',
-            'concepts/roles',
```

Comment:
concepts/roles was deleted in favor of elevating this

other stuff was reordered to annoy me slightly less